Quizzical/src/components/Button.css

```
@@ -0,0 +1,37 @@
+.button {
+  all: unset;
+  padding: 12px;
+  border-radius: 5px;
+}
+
+.button-primary {
+  background-color: #4d5b9e;
+  color: #f5f7fb;
+}
+
+.button-selected {
+  background-color: #d6dbf5;
+}
+
+.button-unselected {
+  background-color: none;
+  border: solid 1px black;
+}
+
+.button-selected-incorrect {
```

```suggestion
.button-incorrect-selected {
```
`selected` is more a variation of `incorrect` rather then the other way around. I'd always order it by specificity (broader things first, more specific things last)